packages/sdk-metrics/src/Descriptor.ts

```diff
@@ -32,19 +32,46 @@ export enum InstrumentType {
 /**
  * An interface describing the instrument.
  */
-export interface InstrumentDescriptor {
+export interface Descriptor {
   readonly name: string;
   readonly description: string;
   readonly unit: string;
-  readonly type: InstrumentType;
   readonly valueType: ValueType;
 }
 
-export function createInstrumentDescriptor(
+/**
+ * An interface describing the instrument.
+ * @deprecated Please use {@link Descriptor} instead.
+ */
+export interface InstrumentDescriptor extends Descriptor {
+  /**
+   * The original instrument's type.
+   * @deprecated Any necessary information about a metric's properties is available in the data point,
+   * see {@link MetricData.dataPointType}
+   */
+  readonly type: InstrumentType;
+}
+
+/**
+ * An interface describing an actual instrument or virtual instrument (created by views).
+ * Only intended for internal use. For exporting descriptor data, use {@link Descriptor} instead.
+ */
+export interface MetricDescriptor extends Descriptor {
+  /**
+   * The original instrument's type.
+   */
+  readonly type: InstrumentType;
+}
+
+export function toExternal(descriptor: MetricDescriptor): InstrumentDescriptor {
+  return descriptor;
+}
+
+export function createDescriptor(
```

Comment:
Since `Descriptor` is a type distinguished from `MetricDescriptor`, we should be clear about what is created here.

```suggestion
export function createMetricDescriptor(
```

Reply:
ah, yes, fixed in [afde29d](https://github.com/open-telemetry/opentelemetry-js/pull/3520/commits/afde29db015198612d7d11a8bc558b84d1db5247) :slightly_smiling_face: